### DISCUSSION TITLE
Why am I unable to issue Updates?

### DISCUSSION THREAD

**Discussion by joshuamcginnis** [dbeaver/dbeaver#33904] [Q&A] [answered]
I recently added a new connection to a hosted Postgres DB and am unable to make any inserts or updates with an unusual error below.

I have confirmed that I can make the updates just fine using `psql`. 

For example, this basic update:
```sql
update company.customer
set outreach_status = 'open'
where external_id = '123'
```

Returns the error:
> SQL Error [23514]: ERROR: new row for relation "customer" violates check constraint "timed_object_created_at_check"

The constrain in the error is:
```sql
timed_object_created_at_check CHECK
((date_part('timezone'::text, created_at) = '0'::double precision));
```

Why is dbeaver interpreting the update as a "new row" and why is the constraint being triggered on an already existing row?

This is not happening with other Postgres DBs I have configured, including local ones running Docker.

This makes me think there is some kind of configuration change I need to make.

Your help would be much appreciated!

**Comment by TB-HunBoka** [comment, marked as answer]
As a workaround can you try to run the update command natively? Not sure if it's available for Posgres DB or not.
Right click -> execute -> execute SQL script natively.

**Reply by joshuamcginnis** [reply]
I suspect that will work. I also found out that to fix the original issue, I needed to set my timezone in the DBeaver config settings to UTC.  But I like the native execution option so thanks for sharing.
